.changeset/new-ghosts-train.md

```diff
@@ -0,0 +1,5 @@
+---
+"@inngest/ai": minor
```

Comment:
Nit, but as this is `0.x.y` atm should this be a `patch`? I think `minor` means a breaking change in `0.x.y`?